### ISSUE TITLE
Forgotten password popup after logged in

### ISSUE THREAD

**Issue by spirillen** [Chocobozzz/PeerTube#6345]
### Describe the current behavior

After haven logged in I get a popup asking if I have forgotten my password...

### Steps to reproduce

1. Login with user/pass
2. You get the popup (some times)

### Describe the expected behavior

This popup should only appears after unsuccesfull login attempt

### Additional information

* PeerTube instance:
  * URL: redacted for privacy
  * Version: `6.0.3`
  * NodeJS version: `v21.7.3`
  * Ffmpeg version: not relevant

* Browser name, version and platforms on which you could reproduce the bug: FF-esr, tor browser
* Link to browser console log if relevant:
* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):

<details><summary>Details</summary>
<p>

```bash
Apr 23 01:10:03 ns3109441 peertube[31217]:     "stack": "TypeError: Cannot read properties of undefined (reading 'replace')\n    at autoblacklistVideos (/var/www/peertube/storage/plugins/node_modules/peertube-plugin-glavliiit/main.js:152:6)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async internalRunHook (file:///var/www/peertube/versions/peertube-v6.0.3/packages/core-utils/dist/plugins/hooks.js:16:19)\n    at async PluginManager.runHook (file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/lib/plugins/plugin-manager.js:166:22)\n    at async autoBlacklistVideoIfNeeded (file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/lib/video-blacklist.js:15:29)\n    at async file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/lib/video-pre-import.js:54:9\n    at async /var/www/peertube/versions/peertube-v6.0.3/node_modules/sequelize/lib/sequelize.js:507:18\n    at async insertFromImportIntoDB (file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/lib/video-pre-import.js:43:25)\n    at async buildYoutubeDLImport (file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/lib/video-pre-import.js:133:25)\n    at async handleYoutubeDlImport (file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/controllers/api/videos/import.js:113:38)",
Apr 23 01:10:03 ns3109441 peertube[31217]:     "message": "Cannot read properties of undefined (reading 'replace')"
Apr 23 01:10:03 ns3109441 peertube[31217]:   }
Apr 23 01:10:03 ns3109441 peertube[31217]: }
Apr 23 01:10:11 ns3109441 peertube[31217]: [Plugin Upload Limits] ACCEPTED No defined limits
Apr 23 01:10:17 ns3109441 peertube[31217]: (node:31217) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
Apr 23 01:10:17 ns3109441 peertube[31217]: (Use `node --trace-deprecation ...` to show where the warning was created)
Apr 23 01:13:00 ns3109441 peertube[31217]: [example.org:443] 2024-04-23 01:13:00.957 error: Cannot run hook filter:video.auto-blacklist.result of plugin glavliiit. {
Apr 23 01:13:00 ns3109441 peertube[31217]:   "err": {
Apr 23 01:13:00 ns3109441 peertube[31217]:     "stack": "TypeError: Cannot read properties of undefined (reading 'replace')\n    at autoblacklistVideos (/var/www/peertube/storage/plugins/node_modules/peertube-plugin-glavliiit/main.js:152:6)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async internalRunHook (file:///var/www/peertube/versions/peertube-v6.0.3/packages/core-utils/dist/plugins/hooks.js:16:19)\n    at async PluginManager.runHook (file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/lib/plugins/plugin-manager.js:166:22)\n    at async autoBlacklistVideoIfNeeded (file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/lib/video-blacklist.js:15:29)\n    at async file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/controllers/api/videos/update.js:104:13\n    at async /var/www/peertube/versions/peertube-v6.0.3/node_modules/sequelize/lib/sequelize.js:507:18\n    at async updateVideo (file:///var/www/peertube/versions/peertube-v6.0.3/dist/core/controllers/api/videos/update.js:43:67)",
Apr 23 01:13:00 ns3109441 peertube[31217]:     "message": "Cannot read properties of undefined (reading 'replace')"
Apr 23 01:13:00 ns3109441 peertube[31217]:   }
Apr 23 01:13:00 ns3109441 peertube[31217]: }
Apr 23 01:13:00 ns3109441 peertube[31217]: [example.org:443] 2024-04-23 01:13:00.966 error: Cannot run hook action:api.video.updated of plugin chapters. {
Apr 23 01:13:00 ns3109441 peertube[31217]:   "err": {
Apr 23 01:13:00 ns3109441 peertube[31217]:     "stack": "TypeError: Cannot read properties of null (reading 'trim')\n    at parseTableOfContents (/var/www/peertube/storage/plugins/node_modules/peertube-plugin-chapters/dist/shared/common.js:21:18)\n    at Object.<anonymous> (/var/www/peertube/storage/plugins/node_modules/peertube-plugin-chapters/dist/server/main.js:37:64)\n    at Generator.next (<anonymous>)\n    at fulfilled (/var/www/peertube/storage/plugins/node_modules/peertube-plugin-chapters/dist/server/main.js:5:58)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)",
Apr 23 01:13:00 ns3109441 peertube[31217]:     "message": "Cannot read properties of null (reading 'trim')"
Apr 23 01:13:00 ns3109441 peertube[31217]:   }
Apr 23 01:13:00 ns3109441 peertube[31217]: }
```

</p>
</details>

**Comment by spirillen** [comment]
Was lucky to reproduce the issue

![image](https://github.com/Chocobozzz/PeerTube/assets/44526987/41a52ea1-2a56-4e3c-b327-ec317e75efc2)

**Comment by Chocobozzz** [comment]
Hi,

Thanks for the issue that should be fixed in the next stable release (6.1)
